### PR TITLE
New and improved holy grail algorithm

### PR DESCRIPTION
**Pull request by profxj**
Developed first for LRIS R900 but tested on others.

### CODE REVIEW

File: arclines/holy/grail.py [hunk 1]
```diff
@@ -185,17 +233,20 @@ def semi_brute(spec, lines, wv_cen, disp, siglev=20., min_ampl=300.,
             if np.min(np.abs(line_lists['wave'][NIST_lines]-idwv)) > 0.01:
                 imsk[kk] = False
         ifit = ifit[imsk]
+        # Allow for weaker lines in the fit
+        all_tcent, weak_cut_tcent, icut = arch_utils.arc_lines_from_spec(spec, min_ampl=300.)
```
Comment (rcooke-ast) [inline arclines/holy/grail.py]: I have a feeling that the second argument should be:  `min_ampl=min_ampl`

**Comment by rcooke-ast** [comment]
Looks good otherwise - feel free to merge when ready!

File: arclines/holy/grail.py (hunk 1 above)
Comment (profxj) [inline arclines/holy/grail.py]: Am actually going to use lowest_amp here.

**Comment by profxj** [comment]
Merging.

**Pull request merged**